src/Akka.Cluster.Hosting/AkkaClusterHostingExtensions.cs

```diff
@@ -696,16 +696,26 @@ public static AkkaConfigurationBuilder WithClustering(
         /// which will return `Unhealthy` until we have successfully joined a cluster. Used to prevent nodes
         /// from accepting load-balancer traffic until we have access to the cluster.
         /// </summary>
+        /// <param name="builder">The builder instance being configured.</param>
+        /// <param name="failureStatus">
+        /// The <see cref="HealthStatus"/> that should be reported upon failure of the health check. If the provided value
+        /// is <c>null</c>, then <see cref="HealthStatus.Unhealthy"/> will be reported.
+        /// </param>
+        /// <param name="tags">A list of tags that can be used for filtering health checks.</param>
         /// <remarks>
         /// If you need to customize the readiness check, you can use <see cref="AkkaConfigurationBuilder.WithHealthCheck(AkkaHealthCheckRegistration)"/> to
         /// register your own <see cref="AkkaHealthCheckRegistration"/> with the <see cref="AkkaClusterReadinessCheck"/>.
         /// </remarks>
         public static AkkaConfigurationBuilder WithAkkaClusterReadinessCheck(
-            this AkkaConfigurationBuilder builder)
```

Comment:
Bulk of the changes are this: make `failureStatus` and `tags` easily customizable on all health checks, since you need that for configuring things like separate liveness / readiness endpoints.